tests/test_chat.py

```diff
@@ -149,3 +150,49 @@ def test_merge_lora_if_needed(mocked_merge_lora, mocked_input, fake_checkpoint_d
 
     assert re.match(r".*Merging LoRA weights with the base model\..*", out.getvalue(), re.DOTALL)
     mocked_merge_lora.assert_called_once()
+
+
+import io
+from unittest.mock import Mock, patch
+from contextlib import redirect_stdout
+
+import litgpt
+from litgpt.utils import auto_download_checkpoint
+
+
+prompt = "Hello world!"
+expected_output_part = "def reverse_string(s):"
+model_name = "microsoft/phi-2"
+
+def test_litgpt_chat_endtoend():
+    from litgpt.chat.base import main
+
+    checkpoint_dir = auto_download_checkpoint(model_name, access_token="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")
```

Comment:
```suggestion
    checkpoint_dir = auto_download_checkpoint(model_name)
```


I think you don't need this because we have a access token in the CI environment. I would avoid exposing tokens in the tests. The download function already does an

```
if access_token is None:
    access_token = os.getenv("HF_TOKEN")
```
internall